spec/mailers/previews/devise_custom_mailer_preview.rb

```diff
@@ -2,7 +2,7 @@
 
 # list these mailer preview links with this URL:
 # http://localhost:3001/rails/mailers/devise_custom_mailer
-
+#
```

Comment:
Was this intentional?

Reply:
rubocop complained about the blank line there. :-)

Reply:
You could just remove the line 😂